src/logger/Logger.js

```diff
@@ -0,0 +1,49 @@
+type LoggerOptions = {
+    onMetricLog?: Function
+};
+
+type MetricData = {
+    component: string,
+    message: string,
+    value?: string | number,
+    timestamp: string,
+    session_id: string,
```

Comment:
`sessionId`?

Reply:
Ah, have to lowercase to follow schema validation

Reply:
I think we'll want it back to `session_id`. The analytics repo standardizes on snake_case